data/desktop-style-ids.txt

```diff
@@ -1,20 +1,21 @@
 # List of recognized GUI environments
 # ID            Human-readable name
 #
-cinnamon    Cinnamon
-dde         Deepin
-ede         EDE
-endless     Endless
-gnome       GNOME
-gnome:dark  GNOME (Dark)
-lxde        LXDE
-lxqt        LXQt
-macos       macOS
-mate        Mate
-pantheon    Pantheon
-plasma      KDE Plasma
-razor       Razor
-rox         Rox
-unity       Unity
-windows     Microsoft Windows
-xfce        Xfce
+cinnamon      Cinnamon
+dde           Deepin
+ede           EDE
+endless       Endless
+gnome         GNOME
+gnome:dark    GNOME (Dark)
+lxde          LXDE
+lxqt          LXQt
+macos         macOS
+mate          Mate
+pantheon      Pantheon
+plasma        KDE Plasma
+plasma-mobile KDE Plasma Mobile
```

Comment:
```suggestion
plasma-mobile Plasma Mobile
```

While for Plasma desktop we often add "KDE" in front, for Plasma Mobile we don't do that anywhere.